src/bus.rs

```diff
@@ -281,6 +281,7 @@ impl<B: UsbBus> UsbBusAllocator<B> {
 
 /// A handle for a USB interface that contains its number.
 #[derive(Copy, Clone, Eq, PartialEq)]
+#[cfg_attr(feature = "defmt-impl", derive(defmt::Format))]
```

Comment:
All of the code blocks use feature `defmt-impl` instead of just `defmt`

Reply:
Added another cargo check with the defmt feature.

Reply:
Good catch! It's been a while since I wrote the PR.